api/src/main/java/jakarta/enterprise/inject/spi/Prioritized.java

```diff
@@ -22,8 +22,8 @@
  * <p>
  * For instance {@link ObserverMethod} extends this interface to set the observer priority.
  *
- * A custom alternative {@link Bean}, {@link Interceptor} or {@link Decorator} may implement this interface to be activated
- * with a given priority
+ * A custom alternative or reserve {@link Bean}, {@link Interceptor} or {@link Decorator} may implement
+ * this interface to be activated with a given priority
```

Comment:
Unrelated to the content of this PR but perhaps we should replace `activated` for `enabled`?
That's the term we are using throughout the spec.

Reply:
Good point.